src/community-guides/tips-for-succeeding-at-hackathons.md

```diff
@@ -0,0 +1,5 @@
+1. Build a balanced team with both coders and presenters. Aim for diversity in skills, such as 2 developers and 2 non-developers. Include roles like UI/UX designers and domain experts if relevant. (Anecdote: Won 4 Mumbai-level hackathons with a team of 2 devs and 2 non-devs.)
+2. Select a strong problem statement (PS). Choose one that's likely to reach the final round and can be explained easily. Look for PS with real-world applications but room for improvement. Research experimental PS thoroughly. Treat optional features as mandatory for a winning entry.
```

Comment:
Possible to add sub-points here about a generic process for identifying strong problem statements? If possible, take examples from past organized hackathons and give insights on which is a strong statement with reason.

Reply:
Yes, planning to add subpoints, this was the overall flow of article.